### DISCUSSION TITLE
YimMenu being sold for $29.99

### DISCUSSION THREAD

**Discussion by ClapperJuan** [YimMenu/YimMenu#2721] [General]
Not sure if you can do anything or if your even intrested in this but a website called [1cheats](https://1cheats.com/store/product/4-embrace-menu-lifetime-license/) is selling YimMenu for $29.99 and all they have done is called the menu "Embrace Menu" on the webiste without even changing anything in the menu it self. They have screenshots and a video showing Yimmenu being used but still sell it for $29.99 without even stating that its a open source free to use menu.

**Comment by Yimura** [comment]
These things happen and will keep on happening, all you can do is spread the word and tarnish the reputation of the website.

**Comment by D-Borkan** [comment]
They exploit people for money and instead they can create a modified version and publish it for free for the benefit of everyone
